template/$PROJECT_NAME$/config/prod.exs

```diff
@@ -72,10 +72,14 @@ config :guardian, Guardian,
 config :<%= @project_name %>, <%= @project_name_camel_case %>.Web.Endpoint,
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
+<%= if @is_heroku? do %>
 # Configure your database
 config :<%= @project_name %>, <%= @project_name_camel_case %>.Repo,
   adapter: Ecto.Adapters.Postgres,
+  url: System.get_env("DATABASE_URL"),
```

Comment:
Going to need this environment variable in the generated README.md (not the one on this project).